server/utils/compileEmailTemplate.js

```diff
@@ -1,10 +1,16 @@
+import { readFileSync } from 'fs';
+import memoize from 'lodash/memoize';
 import flow from 'lodash/fp/flow';
 import template from 'lodash/template';
 import juice from 'juice';
 
-const compileEmailTemplate = flow(
-  juice,
-  template
+const readFile = (path) => readFileSync(path, 'utf8');
```

Comment:
@jmike i wasn't 100% percent sure if i should write this differently (maybe with an apply?) i thought this was clean enough.

Reply:
It's cool what you did here - FP all the way!

There might be a better way to do this using [partialRight()](https://lodash.com/docs/4.17.11#partialRight), e.g.

```javascript
const compileEmailTemplate = memoize(
  flow(
    partialRight(readFileSync, 'utf8'),
    juice,
    template
  )
);
```